sound/soc/sof/control.c

```diff
@@ -69,7 +69,7 @@ static void snd_sof_refresh_control(struct snd_sof_control *scontrol)
 {
 	struct sof_ipc_ctrl_data *cdata = scontrol->control_data;
 	struct snd_soc_component *scomp = scontrol->scomp;
-	enum sof_ipc_ctrl_type ctrl_type;
+	u32 ipc_cmd;
```

Comment:
missing space in commit title: ASoC: SOF: control: Fix variable type in snd_sof_refresh_control()

Reply:
OK.